makefile

```diff
@@ -18,6 +18,11 @@ include make/libraries
 -include $(HOME)/.config/stan/make.local  # define local variables
 -include make/local                       # overwrite local variables
 
+# TODO(Steve): This was in defaults, but make/local is not called till later
```

Comment:
That's right. The pull request I'm working on for make should address this issue. It's just tricky to fix the whole build process in one go, which is why it's taking a while.